util/unittests/test_process.py

```diff
@@ -169,7 +169,7 @@ def test02_is_process_running(self):
                                                          self.ssh_client),
                          'Testing false case for invalid process id. Remote')
 
-    def test03_is_process_running(self):
+    def test_is_process_running(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_is_process_running' from line 161

<!-- Comment by [SideCI](https://sideci.com) -->